website/src/styles/_installation.scss

```diff
@@ -0,0 +1,80 @@
+.installation-body {
+    display: flex;
+    justify-content: center;
+}
+
+.installation {
+    position: relative;
+    border: 1px solid var(--sl-color-gray-5);
+    overflow: hidden;
+    margin-top: 9.5rem;
+    width: 80vw;
+    display: flex;
+    gap: 2.5rem;
+    background:
+        radial-gradient(hsla(var(--sl-hue-red), 68%, 52%, 16%), transparent 40%) no-repeat -60vw -50vh / 105vw 200vh,
+        radial-gradient(var(--sl-color-blue-low), transparent 40%) no-repeat 30vw -100vh / 105vw 200vh;
+}
+
+.installation-left {
+    padding: 2.5rem 2.5rem 4.5rem;
+}
+
+.installation-left>a>img {
+    margin: 0;
+}
+
+.installation-left>a {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.installation-head {
+    font-size: 3.1rem;
+    font-weight: bold;
+}
+
+.installation-des {
+    max-width: 25rem;
+}
+
+.installation-ext,
+.installation-biome {
+    display: flex;
+    margin-top: 1rem;
+    align-items: center;
+    text-decoration: none;
+    padding: 0.5rem 1rem;
+    border: 1px solid var(--sl-color-gray-5);
+    border-radius: 0.5rem;
+    cursor: pointer;
+}
+
+.installation-ext:hover,
+.installation-biome:hover {
+    background: hsla(var(--sl-hue-blue), 68%, 52%, 16%);
+    text-decoration: underline;
+}
+
+.ext-img {
+    position: absolute;
+    height: 24rem;
+    width: 31rem;
+    border-top-left-radius: 0.6rem;
+    background-repeat: no-repeat;
+    background-image: url("../assets/svg/vscode-ext.png");
+    background-size: 330%;
+    right: -7.2rem;
+    bottom: -1.8rem;
+}
+
+@media only screen and (max-width: 1080px) {
```

Comment:
Why 1080? The other media queries have a different threshold.

Maybe it's better to use the mixin: https://github.com/biomejs/biome/blob/28f2de482625e2ce650b407637981720d1298df9/website/src/styles/_global.scss#L129

Reply:
I use this because we hide the vscode image extension on 1080.

Reply:
It's worth adding a comment that explains it :)